test/uk/gov/hmrc/trusts/services/ValidationServiceSpec.scala

```diff
@@ -52,6 +51,13 @@ class ValidationServiceSpec extends BaseSpec {
         validator.validate[ExistingTrustCheckRequest](jsonString).isLeft mustBe true
       }
     }
-  }
+    "return a list of validaton errors " when {
+      "json request is valid but failed in business rules. " in {
+        val jsonString = JsonUtils.getJsonFromFile("trust-business-validation-fail.json")
+        validator.validate[Registration](jsonString).isLeft mustBe true
+        validator.validate[Registration](jsonString).left.toOption.get.size mustBe 3
```

Comment:
@manishsw Are you aware of http://www.scalatest.org/user_guide/other_goodies#eitherValues to test Eithers?

Reply:
updated.